packages/test/vite.config.ts

```diff
@@ -1,19 +1,29 @@
 // Copyright © 2023 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
 
+import path from "path"
 import { defineConfig } from "vite"
 import dts from "vite-plugin-dts"
-import path from "path"
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [
     dts({
+      root: path.resolve(__dirname, "../../src/test"),
```

Comment:
Why is this necessary? Looks a bit odd having the "root" outside the current directory.